openshift2nulecule/cli/main.py

```diff
@@ -145,14 +145,26 @@ def run(self):
             nulecule_artifacts = []
 
             os.makedirs(path)
-            filepath = os.path.join(path, "artifacts.json")
-            nulecule_artifacts.append("file://{}".format(os.path.relpath(
-                filepath, nulecule_dir)))
-            anymarkup.serialize_file(exported_project[provider].artifacts,
-                                     filepath,
-                                     format="json")
+            # create artifact files
+            for artifact in exported_project[provider].artifacts:
+                if "name" in artifact["metadata"]:
+                    name = artifact["metadata"]["name"]
+                else:
+                    name = "unknown"
+                kind = artifact["kind"]
+                filename = "{}-{}.json".format(name, kind)
+                filepath = os.path.join(artifacts_dir, filename)
+                if os.path.exists(filepath):
+                    filepath = utils.get_new_name(filepath)
+                    print filepath
```

Comment:
Aj, good catch. This is just debug print. This shouldn't be there at all :laughing: